charts/drax/values.yaml

```diff
@@ -585,17 +585,9 @@ grafana:
 loki-stack:
   enabled: true
   promtail:
-    enabled: false
```

Comment:
Given the conscious change from fluent-bit, should we explicitly set this to `true`?

Reply:
I don't know, it's not that it would be really visible inside the values file. In the end the commit message will be included in the release notes, so it would be nicely documented there at least (and I think that's enough)...